### PR TITLE
PTW-25: Update(fix) to pagination support and adds documentation

### PR DESCRIPTION
**Pull request by peterchallett**

### CODE REVIEW

File: lib/MwbExporter/Formatter/Doctrine2/Annotation/OnCorps/ApiPlatformResourceAnnotations.php [hunk 1]
```diff
@@ -39,23 +40,54 @@ public function buildAnnotations(Table $table) : self
...
+        if(isset($settings['max_items']) &&  filter_var($settings['max_items'], FILTER_VALIDATE_INT)){
+            $itemsPerPage = (int)trim($settings['max_items']);
+            $properties[] = ' "maximum_items_per_page"=' . $itemsPerPage;
+        }
+
+        if(isset($settings['client_control'])) {
+            $properties[] = ' "pagination_client_enabled"=' . ($settings['client_control'] === 'true' ? 'true' : 'false');
+            $properties[] = ' "client_items_per_page"=' . ($settings['client_control'] === 'true' ? 'true' : 'false');
+        } else {
+            //User has put in a pagination comment but not been explicit so assume thay want pagination on.
```
Comment (TomEdwards) [inline lib/MwbExporter/Formatter/Doctrine2/Annotation/OnCorps/ApiPlatformResourceAnnotations.php]: is this comment correct? Looks like a reasonable aspiration but implementation appears to be disable

**Review by TomEdwards — CHANGES_REQUESTED** [review]
One comment but otherwise looks good.

File: lib/MwbExporter/Formatter/Doctrine2/Annotation/OnCorps/ApiPlatformResourceAnnotations.php (hunk 1 above)
Comment (peterchallett) [reply lib/MwbExporter/Formatter/Doctrine2/Annotation/OnCorps/ApiPlatformResourceAnnotations.php]: Good catch - copy and paste error. Will fix intent.

**Review by peterchallett — COMMENTED** [review]
Is this better?

**Review by TomEdwards — APPROVED** [review]
Looks good to me

**Pull request merged**